journal/modernized/08-chapter-7.adoc

```diff
@@ -224,10 +224,10 @@ And thus the warning before given them was fulfilled.
 
 In the year 1609, George Fox came to Ireland,
 and several Friends in the ministry with him.
-Me settled men`'s and women`'s meetings among Friends throughout the nation,
+He settled men`'s and women`'s meetings among Friends throughout the nation,
 that faithful men and women should take care in the government of church affairs,
 among our own Society, which meetings are of great service.
-I was much cased by them, as I told George Fox at that time,
+I was much eased by it, as I told George Fox at that time,
```

Comment:
So when I went to look up "cased" in the PDF, I noticed "them " is actually supposed to be "it". Just wanted you to know why there was such a big change. Also, I accidentally tapped some keys where I could use emojis but i have no idea how I did it! Can you tell me how?? :)

Reply:
👍 Thanks!!